### PR TITLE
docs: add "alumni" section to README

### PR DESCRIPTION
**Pull request by bajtos**
I hope I listed everybody who used to be a maintainer at some point in the past 🤞 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by bajtos** [comment]
Updating the Markdown table was a bit of PITA, maybe we should switch to HTML syntax for easier maintenance? Anyhow, such change is out of scope of this PR.

**Comment by bajtos** [comment]
We will need to update CODEOWNERS file too, but I don't have bandwidth to discuss new owners.

File: package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
     "@commitlint/cli": "^9.0.1",
     "@commitlint/config-conventional": "^9.0.1",
     "@commitlint/travis-cli": "^9.0.1",
+    "@swc/core": "^1.2.6",
```
Comment (dhmlau) [inline package.json:17]: Not sure if we need the added dependencies. Or am i missing something?

Comment (raymondfeng) [reply package.json:17]: That's probably by mistake.

Comment (raymondfeng) [reply package.json]: I reverted such changes.

File: README.md [hunk 2]
```diff
@@ -98,28 +98,37 @@ See [SECURITY.md](SECURITY.md) for more details.
...
+
+- [@ritch](http://github.com/ritch)
+- [@superkhau](https://github.com/superkhau)
+- [@rashmihunt](https://github.com/rashmihunt)
+- [@kjdelisle](https://github.com/kjdelisle)
+- [@virkt25](https://github.com/virkt25)
+- [@shimks](https://github.com/shimks)
+- [@nabdelgadir](https://github.com/nabdelgadir)
+- [@deepakrkris](https://github.com/deepakrkris)
+
```
Comment (agnes512) [inline README.md:131]: - [Biniam](https://github.com/b-admike)

:D

Comment (raymondfeng) [reply README.md:131]: Do you want to fix the PR? I assume @bajtos is on vacation for some time.

Comment (agnes512) [reply README.md:132]: Fixed!

**Pull request merged**